assembly/src/assembler/context.rs

```diff
@@ -373,6 +399,16 @@ impl ModuleContext {
             .map(|proc| proc.inner())
     }
 
+    /// Returns the reference to the procedures stack.
+    pub fn proc_stack(&self) -> &Vec<ProcedureContext> {
+        &self.proc_stack
+    }
+
+    /// Returns the reference to the compiled procedures.
+    pub fn compiled_procs(&self) -> &Vec<NamedProcedure> {
+        &self.compiled_procs
+    }
+
```

Comment:
I think we can avoid introducing these methods. Instead, we might need to add something similar to `get_imported_procedure_name()` function I mentioned in one of the previous comments.